tags.lisp

```diff
@@ -283,7 +296,27 @@
       (:track :kind :src :srclang :label :default)
       (:ul :type)
       (:video :autoplay :preload :controls :loop :poster
-              :mediagroup :muted :src :crossorigin)))
+              :mediagroup :muted :src :crossorigin)
+      ;; SVG attributes
+      (:circle :id :class :style :transform :cx :cy :r)
+      (:ellipse :id :class :style :transform :cx :cy :rx :ry)
+      (:line :id :class :style :transform :x1 :y1 :x2 :y2)
+      (:path :id :class :style :transform :d)
+      (:polygon :id :class :style :transform :points)
+      (:polyline :id :class :style :transform :points)
+      (:rect :id :class :style :transform :x :y :width :height :rx :ry)
+      (:mesh :id :class :style :transform)
+      (:text :id :class :style :transform :x :y :dx :dy :rotate :textLength :lengthAdjust)
+      (:tspan :id :class :style :transform :x :y :dx :dy :rotate)
+      (:textPath :id :class :style :transform :href :startOffset :method :spacing :side :path)
+      (:defs :id :class :style :transform)
+      (:g :id :class :style :transform)
+      (:image :id :class :style :transform :href :x :y :width :height :preserveAspectRatio :externalResourceRequired :corssorigin )
+      (:svg :id :class :style :transform :x :y :width :height :viewBox :preserveAspectRatio
+            :linecap :id :class :style :transform :fill :stroke :stroke-width :stroke-linecap :stroke-linejoin)
```

Comment:
The duplicate id here breaks the pipeline.

Reply:
The duplicate `:id` `:class` `:style` `:transform` have been removed from the `:svg` element.